src/toil/test/jobStores/jobStoreTest.py

```diff
@@ -423,6 +424,58 @@ def assertUrl(self, url):
                     urllib2.urlopen(urllib2.Request(url))
                 except:
                     self.fail()
+                    
+        def testCleanCache(self):
+            # Make a bunch of jobs
+            master = self.master
+
+            # Create parent job
+            rootJob = master.create('rootjob', 12, 34, 35)
+            
+            # Create a bunch of child jobs
+            for i in range(100):
+                child = master.create("child%s" % i, 23, 45, 46, 1)
+                rootJob.stack.append(((child.jobStoreID, 23, 45, 46, 1),))
+            master.update(rootJob)
+            
+            # See how long it takes to clean with no cache
+            noCacheStart = time.time()
+            master.clean(rootJob)
+            noCacheEnd = time.time()
+            
+            noCacheTime = noCacheEnd - noCacheStart
+            
+            # See how long it takes to clean with cache
+            jobCache = {jobWrapper.jobStoreID: jobWrapper
+                        for jobWrapper in master.jobs()}
+            cacheStart = time.time()
+            master.clean(rootJob, jobCache=jobCache)
+            cacheEnd = time.time()
+            
+            cacheTime = cacheEnd - cacheStart
+            
+            print("Without cache: %s With cache: %s" % (str(noCacheTime), str(cacheTime)))
```

Comment:
It seems perfectly happy with the syntax without that, though it might be parsed as a print statement and then a parenthesized string.

Do we use that elsewhere in Toil?